pipeline/scripts/average_rating.py

```diff
@@ -0,0 +1,37 @@
+import sys
+
+sys.path.insert(0, "/workspace/butterfly/backend/utils/")
```

Comment:
This path will only work on GitPod, since it assumes `butterfly/` will be inside a `workspace/` folder.

You can do this instead:

```python
import sys

sys.path.append("./")
```

This is what the other scripts in this folder do.